TF introduced a change pinning versions to fixed major.minor or even patch versions
Loosen those a bit so we can build it with our versions.
See https://github.com/tensorflow/tensorflow/issues/44654

Author: Alexander Grund (TU Dresden)

diff --git a/tensorflow/tools/pip_package/setup.py b/tensorflow/tools/pip_package/setup.py
index 354657a7bfc..d4c880402f7 100644
--- a/tensorflow/tools/pip_package/setup.py
+++ b/tensorflow/tools/pip_package/setup.py
@@ -78,20 +78,20 @@ REQUIRED_PACKAGES = [
     # NOTE: As numpy has releases that break semver guarantees and several other
     # deps depend on numpy without an upper bound, we must install numpy before
     # everything else.
-    'numpy ~= 1.19.2',
+    'numpy >= 1.17.3',
     # Install other dependencies
     'absl-py ~= 0.10',
     'astunparse ~= 1.6.3',
     'flatbuffers ~= 1.12.0',
     'google_pasta ~= 0.2',
-    'h5py ~= 3.1.0',
+    'h5py >= 2.10.0',
     'keras_preprocessing ~= 1.1.2',
     'opt_einsum ~= 3.3.0',
     'protobuf >= 3.9.2',
-    'six ~= 1.15.0',
+    'six >= 1.12.0',
     'termcolor ~= 1.1.0',
     'typing_extensions ~= 3.7.4',
-    'wheel ~= 0.35',
+    'wheel >= 0.33',
     'wrapt ~= 1.12.1',
     # These packages need to be pinned exactly as newer versions are
     # incompatible with the rest of the ecosystem
@@ -102,9 +102,6 @@ REQUIRED_PACKAGES = [
     # When updating these, please also update the nightly versions below
     'tensorboard ~= 2.5',
     'tensorflow-estimator >= 2.5.0rc0 , < 2.6.0',
-    # TODO(scottzhu): OSS keras hasn't been formally released yet.
-    # Use keras-nightly at the moment.
-    'keras-nightly ~= 2.5.0.dev',
 ]
 
 
